src/streamlink/plugins/vk.py

```diff
@@ -21,10 +21,10 @@
 
 
 @pluginmatcher(re.compile(
-    r"https?://(?:\w+\.)?vk\.com/videos?(?:\?z=video)?(?P<video_id>-?\d+_\d+)",
+    r"https?://(?:\w+\.)?vk(?:\.com|.ru)/videos?(?:\?z=video)?(?P<video_id>-?\d+_\d+)",
```

Comment:
```suggestion
    r"https?://(?:\w+\.)?vk\.(?:com|ru)/videos?(?:\?z=video)?(?P<video_id>-?\d+_\d+)",
```

The dot needs to be escaped and can be moved out of the non-capture group